<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="Vijender Singh Publications">
    <meta name="author" content="Vijender Singh">
    <title>Publications</title>
    <!-- font icons -->
    <link rel="stylesheet" href="assets/vendors/themify-icons/css/themify-icons.css">
    <!-- Bootstrap + main styles -->
	<link rel="stylesheet" href="assets/css/publications.css">
</head>
<body data-spy="scroll" data-target=".navbar" data-offset="40" id="home">
    <header class="header">
        <div class="container">
            <ul class="social-icons pt-3">
                <li class="social-item"><a target="_blank" class="social-link text-light" href="https://www.instagram.com"><i class="ti-linkedin" aria-hidden="true"></i></a></li>
                <li class="social-item"><a target="_blank" class="social-link text-light" href="https://www.twitter.com"><i class="ti-twitter" aria-hidden="true"></i></a></li>
                <li class="social-item"><a target="_blank" class="social-link text-light" href="https://www.linkedin.com/in/singhvijender/"><i class="ti-instagram" aria-hidden="true"></i></a></li>
                <li class="social-item"><a target="_blank" class="social-link text-light" href="https://github.com/vijender-singh"><i class="ti-github" aria-hidden="true"></i></a></li>
            </ul>  
            <div class="header-content">
                <h1 class="header-title">Publications</h1>
                <button class="btn btn-primary btn-rounded"><a style="color:white" href="https://github.com/vijender-singh/vijender-singh/blob/main/VijenderSingh_Resume.pdf">Download Resume</a></button>
            </div>
        </div>
    </header>
    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white" data-spy="affix" data-offset-top="510">
        <div class="container">
            <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse mt-sm-20 navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav mr-auto">
                    <li class="nav-item">
                        <a href="index.html" class="nav-link">Home</a>
                    </li>
                    <li class="nav-item">
                        <a href="index.html" class="nav-link">About</a>
                    </li>
                    <li class="nav-item">
                        <a href="#resume" class="nav-link">Resume</a>
                    </li>
                </ul>
                <ul class="navbar-nav ml-auto">
                    <li class="nav-item last-item">
                        <a href="#blog" class="nav-link">Projects</a>
                    </li>
                    <li class="nav-item">
                        <a href="#Publications" class="nav-link">Publications</a>
                    </li>
                    <li class="nav-item">
                        <a href="#Resources" class="nav-link">Resources</a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>


<section class="section" id="blog">
    <div class="container">
        <h2 class="mb-5">Published <span class="text-danger">Projects</span></h2>
        <div class="row">
            <div class="blog-card">
                <div class="img-holder">
                    <img src="assets/imgs/Epigenetics.jpg" alt="Project 1">
                </div>
                <div class="content-holder">
                    <h6 class="title">Cumulative stress, PTSD, and emotion dysregulation during pregnancy and epigenetic age acceleration in Hispanic mothers and their newborn infants</h6>

                    <p class="post-details">
                        <a href="#">Authors: Katrinli S, Smith AK., Drury SS, Covault J, Ford JD, Singh V, Reese B, Johnson A, Scranton V, Fall P, Briggs-Gowan M, Grasso DJ.</a>
                    </p>
                    <p><b>Journal: Epigenetics (Accepted), 2023</b></p>                      
                    <p>Here, we evaluated the associations between PTSD symptoms, maternal DNA methylation (DNAm) GrimAge acceleration, which is an epigenetic predictor of mortality, and gestational epigenetic age acceleration in 89 maternal-neonatal dyads</p>
                    <p><b>Platform: Infinium MethylationEPIC BeadChip</b></p>
                    <p><b>Tools used: R, bioconductor</b></p>
                    <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                    <a class="read-more" href="https://github.com/vijender-singh/EpicArrayAnalysis">Github<i class="ti-angle-double-right"></i></a>
                </div>
            </div><!-- end of blog wrapper -->
            <!-- blog-card -->
            <div class="blog-card">
                <div class="img-holder">
                    <img src="assets/imgs/sciAdvFeb2023.jpg" alt="Project 2">
                </div>
                <div class="content-holder">
                    <h6 class="title">Anti-seed PNAs targeting multiple oncomiRs for brain tumor therapy.</h6>

                    <p class="post-details">
                        <a href="#">Authors: Wang Y, Malik S, Suh HW, Xiao Y, Deng Y, Fan R, Huttner A, Bindra RS, Singh V, Saltzman WM, Bahal R.</a>
                    </p>
                    <p><b>Journal: Science Advances 2023 Feb 10;9(6)</b></p>    
                    <p> Glioblastoma (GBM) is one of the most lethal malignancies with poor survival and high recurrence rates. Here, we aimed to simultaneously target oncomiRs 10b and 21, reported to drive GBM progression and invasiveness.</p>
                    
                    <p><b>Tools used: Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R, DESeq2, Database querying (TCGA)</b></p>
                    <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/36753549/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                    <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                </div>
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/blumlietal.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Acute depletion of the ARID1A subunit of SWI/SNF complexes reveals distinct pathways for activation and repression of transcription">Article TITLE</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Blümli S, Wiechens N, Wu MY, Singh V, Gierlinski M, Schweikert G, Gilbert N, Naughton C, Sundaramoorthy R, Varghese J, Gourlay R, Soares R, Clark D, Owen-Hughes T. </a>
                                </p>
                                <p><b>Journal: Cell Rep. 2021 Nov 2;37(5)</b></p>    
                                <p> The ARID1A subunit of SWI/SNF chromatin remodeling complexes is a potent tumor suppressor. Here, a degron is applied to detect rapid loss of chromatin accessibility at thousands of loci where ARID1A acts to generate accessible minidomains of nucleosomes.</p>
                                
                                <p><b>Tools used:Python, R, Linux, Fastqc, Trimmomatic, HiSAT2, HTSeq-count, MACS2, DiffBind, Bedtools, EdgeR, Deeptools  </b></p>
                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/34731603/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/gallagher.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Complete mitochondrial genome of the Caribbean reef shark, Carcharhinus perezi (Carcharhinformes: Carcharhinidae).</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Gallagher AJ, Shipley ON, Reese B, Singh V. </a>
                                </p>
                                <p><b>Journal: Mitochondrial DNA B Resour. 2021 Aug 19;6(9):2662-2664.</b></p>    
                                <p> We report the mitogenome sequence of the Caribbean reef shark to be 16,709 bp and composed two rRNA genes, 22 tRNA genes, 13 protein-coding genes, 2 non-coding regions; the D-loop control region and the origin of light-strand replication.
                                     We build bayesian inference tree of the phylogenetic relationships among 16 shark species within the Order Carcharhiniformes. </p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, MitoZ, MUSCLE, GetOrganelle </b></p>
                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8381894/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/sutter.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Mesenchyme-specific loss of Dot1L histone methyltransferase leads to skeletal dysplasia phenotype in mice. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Sutter P., Karki S., Crawley I., Singh V., Rowe D., Crocker SJ., Bernt K., Bayarsaihan D., and Guzzo RM.  </a>
                                </p>
                                <p><b>Journal: Bone 2021 Jan; 142:115677.</b></p>    
                                <p> We report a novel skeletal dysplasia phenotype in mice with conditional loss of Disruptor of telomeric silencing 1-like (Dot1L) histone methyltransferase in limb mesenchymal progenitors and downstream descendants. 
                                    RNAseq analysis of Dot1L deficient chondrocytes implicated Dot1L in the regulation of key genes and pathways necessary to promote cell cycle regulation and skeletal growth. 
                                </p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R, DESeq2, GSEA, IPA, Panther</b></p>
                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7744341/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/rahmatpour.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">High gene space divergence contrasts with frozen vegetative architecture in the moss family Funariaceae. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Rahmatpour N, Perera NV, Singh V., Wegrzyn JL, Goffinet B. </a>
                                </p>
                                <p><b>Journal: Mol Phylogenet Evol. 2021 Jan;154</b></p>    
                                <p> We analyzed the vegetative gametophytic transcriptome of F. hygrometrica and P. pyriforme and mapped short reads, transcripts, and proteins to the genome and gene space of P. patens. </p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, Bowtie2, Kallisto, R, Trinity, Transdecoder, EnTAP, BUSCO, RNAQuast, VSEARCH, hmmSCAN</b></p>
                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/32956800/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/perry.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">The Potential Role of Preoperative Pain, Catastrophizing, and Differential Gene Expression on Pain Outcomes after Pediatric Spinal Fusion.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Perry M, Sieberg CB, Young EE, Baumbauer K, Singh V, Wong C, Starkweather A. </a>
                                </p>
                                <p><b>Journal: Pain Manag Nurs. 2021 Feb;22(1):44-49.</b></p>    
                                <p> While adolescent idiopathic scoliosis is not often classified as a painful condition, providers must be cognizant of pre-existing pain and anxiety that 
                                    may precipitate a negative recovery trajectory. Policy and practice change are essential for early identification and subsequent intervention.</p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R, DESeq2, GSEA</b></p>
                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8742610/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/knapp.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Nuclear receptor Ftz-f1 promotes follicle maturation and ovulation partly via bHLH/PAS transcription factor Sim.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Knapp EM., Li W., Singh V., Sun J.</a>
                                </p>
                                <p><b>Journal: eLife 2020; 9: e54568</b></p>    
                                <p> Here, we discover that Ftz-f1, one of the NR5A nuclear receptors in Drosophila, is transiently induced in follicle cells in late stages of oogenesis via ecdysteroid signaling. Genetic disruption of
                                     Ftz-f1 expression prevents follicle cell differentiation into the final maturation stage, which leads to anovulation. </p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R, DESeq2, CUT&RUN, MACS2, MEME2, Bowtie2</b></p>
                                <a class="read-more" href="https://elifesciences.org/articles/54568">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/sharma.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Investigating global gene expression changes in a murine model of cherubism.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Sharma T, Cotney J, Singh V, Sanjay A., Reichenberger EJ., Ueki Y.,</a>
                                </p>
                                <p><b>Journal: Bone. 2020; 135:115315.</b></p>    
                                <p> To develop a deeper understanding for the potential underlying mechanisms for Cherubism, a rare genetic disorder, we compared global 
                                    gene expression changes in hematopoietic and mesenchymal cell populations between cherubism and wild type mice.These studies reveal 
                                    complex changes in gene expression elicited from a cherubic mutation in Sh3bp2 that are informative to the mechanisms responding to inflammatory 
                                    stimuli and repressing osteogenesis. </p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R, DESeq2, GSEA</b></p>
                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/32165349/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/boshans.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">The chromatin environment around interneuron genes in NG2 cells and their direct reprogramming into GABAergic neuron-like cells. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Boshans L., William Wood W., C. Factor C.D., Singh V., Jia Lu J., Zhao C., Mandoiu I., Lu R., Cassacia P., Tesar p., Nishiyama A. </a>
                                </p>
                                <p><b>Journal: Frontiers in Neuroscience 2019; 13: 829.</b></p>    
                                <p> Our bioinformatic analysis of histone posttranslational modifications at interneuron genes in Oligodendrocyte precursor cells (OPCs) revealed that OPCs had significantly fewer bivalent and repressive histone marks 
                                    at interneuron genes compared to astrocytes or fibroblasts.</p>
                                
                                <p><b>Tools used: Python, Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R, Bioconductor, DESeq2, GSEA, GOprofiler, MACS2, Bowtie2</b></p>
                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6694778/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/subbu.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title"> Structural reorganization of the chromatin remodeling enzyme Chd1 upon engagement with nucleosomes. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Sundaramoorthy R, Hughes AL, Singh V, Wiechens N, Ryan DP, El-Mkami H,Petoukhov M, Svergun DI, Treutlein B, Quack S, Fischer M, Michaelis J, Böttcher B, Norman DG, Owen-Hughes T.</a>
                                </p>
                                <p><b>Journal: Elife. 2017 Mar 23;6.</b></p>    
                                <p> We identified that the orientation of the DNA-binding domain, in Chd1, is mediated by sequences in the N-terminus and mutations to this part of the protein have positive and negative effects on Chd1 activity. 
                                    These observations indicate that the unfavorable alignment of C-terminal DNA-binding region in solution contributes to an auto-inhibited state.</p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, Python, MACS2</b></p>
                                <a class="read-more" href="https://elifesciences.org/articles/22510">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/hauer.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Histone degradation in response to DNA damage enhances chromatin dynamics and recombination rates.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Hauer MH, Seeber A, Singh V, Thierry R, Sack R, Amitai A, Kryzhanovska M, Eglinger J, Holcman D, Owen-Hughes T, Gasser SM. </a>
                                </p>
                                <p><b>Journal: Nat Struct Mol Biol. 2017 Feb;24(2):99-107.</b></p>    
                                <p>  Here we evaluated whether histones modification and eviction at sites of DNA breaks is localised to these sites or is spread genome wide. we propose that a 
                                    generalized reduction in nucleosome occupancy is an integral part of the DNA damage response in yeast that provides mechanisms for enhanced chromatin mobility 
                                    and homology search. </p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, Bowtie2, MACS2, Bedtools, Python</b></p>
                                <a class="read-more" href="https://www.nature.com/articles/nsmb.3347">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/dickerson.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">High resolution imaging reveals heterogeneity in chromatin states between cells that is not inherited through cell division. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Dickerson D, Gierliński M, Singh V, Kitamura E, Ball G, Tanaka TU, Owen-Hughes T. </a>
                                </p>
                                <p><b>Journal: BMC Cell Biol. 2016 Sep 8;17(1):33.</b></p>    
                                <p> Genomes of eukaryotes exist as chromatin, and it is known that different chromatin states can influence gene regulation. Chromatin is not a static structure, 
                                    but is known to be dynamic and vary between cells. In order to monitor the organisation of chromatin in live cells we have engineered fluorescent fusion proteins which recognize specific operator sequences to 
                                    tag pairs of syntenic gene loci. The separation of these loci was then tracked in three dimensions over time using fluorescence microscopy.</p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, Bowtie2, Python, MACS2</b></p>
                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5016949/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/schurch.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">How many biological replicates are needed in an RNA-seq experiment and which differential expression tool should you use? </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Schurch NJ, Schofield P, Gierliński M, Cole C, Sherstnev A, Singh V, Wrobel N, Gharbi K, Simpson GG, Owen-Hughes T, Blaxter M, Barton GJ</a>
                                </p>
                                <p><b>Journal: RNA. 2016 Jun;22(6):839-51. </b></p>    
                                <p> An RNA-seq experiment with 48 biological replicates in each of two conditions was performed to answer these questions and provide guidelines for experimental design. </p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, STAR, FeatureCount, R, DESeq2, EdgeR, Limma, etc</b></p>
                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/27022035/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/wiechens.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">The Chromatin Remodelling Enzymes SNF2H and SNF2L Position Nucleosomes adjacent to CTCF and Other Transcription Factors. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Wiechens N, Singh V, Gkikopoulos T, Schofield P, Rocha S, Owen-Hughes T. </a>
                                </p>
                                <p><b>Journal: PLoS Genet. 2016 Mar 28;12(3). </b></p>    
                                <p> Here we have sought to investigate which chromatin remodelling enzymes are responsible for this. We find that the ATP-dependent chromatin remodelling enzyme SNF2H plays a major role organising arrays 
                                    of nucleosomes adjacent to the binding sites for the architectural transcription factor CTCF sites and acts to promote CTCF binding.</p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R, DESeq2, Python, MACS2, bedtools</b></p>
                                <a class="read-more" href="FILL THIS UP">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/gierlinski.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Article Statistical models for RNA-seq data derived from a two-condition 48-replicate experiment.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Gierliński M, Cole C, Schofield P, Schurch NJ, Sherstnev A, Singh V, Wrobel N, Gharbi K, Simpson G, Owen-Hughes T, Blaxter M, Barton GJ.</a>
                                </p>
                                <p><b>Journal: Bioinformatics. 2015 Nov 15;31(22):3625-30. </b></p>    
                                <p> A 48-replicate RNA-seq experiment in yeast was performed and data tested against theoretical models. The observed gene read counts were consistent with both log-normal 
                                    and negative binomial distributions, while the mean-variance relation followed the line of constant dispersion parameter of ∼0.01. The high-replicate data also allowed 
                                    for strict quality control and screening of ‘bad’ replicates, which can drastically affect the gene read-count distribution.</p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, HiSAT2, HTSeq-count, R</b></p>
                                <a class="read-more" href="https://academic.oup.com/bioinformatics/article/31/22/3625/240923">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                <a class="read-more" href="https://github.com/vijender-singh">Github<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/allan.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">A comparison of in vitro nucleosome positioning mapped with chicken, frog and a variety of yeast core histones. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Allan J, Fraser RM, Owen-Hughes T, Docherty K, Singh V.</a>
                                </p>
                                <p><b>Journal: J Mol Biol. 2013 Nov 15;425(22):4206-22. </b></p>    
                                <p> Using high-throughput sequencing, we have mapped sequence-directed nucleosome positioning in vitro on four plasmid DNAs containing DNA fragments derived from the genomes of sheep, drosophila, human and yeast. </p>
                                
                            
                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3899014/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/singh.gb.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Evolutionary insights into genome-wide nucleosome positioning. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Singh V, Owen-Hughes T. </a>
                                </p>
                                <p><b>Journal: Genome Biol. 2012;13(9):170.</b></p>    
                                <p> Review Article </p>
                                <a class="read-more" href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-9-170">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                               
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/decian.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">ATP-independent cooperative binding of yeast Isw1a to bare and nucleosomal DNA.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: De Cian A, Praly E, Ding F, Singh V, Lavelle C, Le Cam E, Croquette V, Piétrement O, Bensimon D.</a>
                                </p>
                                <p><b>Journal: PLoS One. 2012;7(2):e31845</b></p>    
                                <p>  Here we use TEM imaging and single molecule manipulation to investigate the interaction between DNA and yeast Isw1a. We show that Isw1a displays a highly cooperative ATP-independent binding to and bridging between DNA segments.</p>

                                <a class="read-more" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0031845">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
        
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/sen.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">UDP-glucose 4, 6-dehydratase activity plays an important role in maintaining cell wall integrity and virulence of Candida albicans. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Sen M, Shah B, Rakshit S, Singh V, Padmanabhan B, Ponnusamy M, Pari K, Vishwakarma R, Nandi D, Sadhale PP. </a>
                                </p>
                                <p><b>Journal: PLoS Pathog. 2011 Nov;7(11):</b></p>    
                                <p> We report here that GAL102 in C. albicans encodes a homolog of dTDP-glucose 4,6-dehydratase, an enzyme that affects cell wall properties as well as virulence of many pathogenic bacteria. We found that GAL102 deletion leads to greater sensitivity to antifungal drugs and cell wall destabilizing agents like Calcofluor white and Congo red. </p>

                                <a class="read-more" href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1002384">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                                
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/trias.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">A role for Snf2-related nucleosome-spacing enzymes in genome-wide nucleosome organization. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Gkikopoulos T, Schofield P, Singh V, Pinskaya M, Mellor J, Smolle M, Workman JL, Barton GJ, Owen-Hughes T</a>
                                </p>
                                <p><b>Journal: Science. 2011 Sep 23;333(6050):1758-60.
                                </b></p>    
                                <p> Here, we show that the combined action of Isw1 and Chd1 nucleosome-spacing enzymes is required to maintain this organization. In the absence of these enzymes, regular positioning of the majority of nucleosomes is lost.</p>
                                
                                <p><b>Tools used: Fastqc, Trimmomatic, Bowtie, Python</b></p>
                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/21940898/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                              
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/ryan.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">The DNA-binding domain of the Chd1 chromatin-remodelling enzyme contains SANT and SLIDE domains.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Ryan DP, Sundaramoorthy R, Martin D, Singh V, Owen-Hughes T. </a>
                                </p>
                                <p><b>Journal: EMBO J. 2011 May 27;30(13):2596-609. </b></p>    
                                <p> Here, we show the DNA-binding domain is required for Saccharomyces cerevisiae Chd1 to bind and remodel nucleosomes. The crystal structure of this domain reveals the presence of structural homology to SANT and SLIDE domains previously identified in ISWI remodelling enzymes.</p>

                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/21623345/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                               
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/trias.cse4.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">The SWI/SNF complex acts to constrain distribution of the centromeric histone variant Cse4. </h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Gkikopoulos T, Singh V, Tsui K, Awad S, Renshaw MJ, Scholfield P, Barton GJ, Nislow C, Tanaka TU, Owen-Hughes T.</a>
                                </p>
                                <p><b>Journal: EMBO J. 2011 May 18;30(10):1919-27. </b></p>    
                                <p> We investigated the role of Swi/Snf in centromeric nucleosome maintainance in yeast.</p>

                                <a class="read-more" href="FILL THIS UP">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                              
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/bowman.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">The histone chaperones Nap1 and Vps75 bind histones H3 and H4 in a tetrameric conformation.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Bowman A, Ward R, Wiechens N, Singh V, El-Mkami H, Norman DG, Owen-Hughes T.</a>
                                </p>
                                <p><b>Journal: Mol Cell. 2011 Feb 18;41(4):398-408.</b></p>    
                                <p> Nucleosome assembly proteins (Nap proteins) represent a distinct class of histone chaperone. Using pulsed electron double resonance (PELDOR) measurements and protein crosslinking, we show that two members of this class, Nap1 and Vps75, bind histones in the tetrameric conformation also observed when they are sequestered within the nucleosome. </p>

                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3093613/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                               
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/singh.fgb.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">The key enzyme in galactose metabolism, UDP-galactose-4-epimerase, affects cell-wall integrity and morphology in Candida albicans even in the absence of galactose.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Singh V, Satheesh SV, Raghavendra ML, Sadhale PP.</a>
                                </p>
                                <p><b>Journal: Fungal Genet Biol. 2007 Jun;44(6):563-74.</b></p>    
                                <p> We showed that S. cerevisiae GAL10 (ScGAL10), essential for survival in the presence of galactose, has not been reported to have defects in the absence of galactose, the C. albicans homolog shows these phenotypes during growth in the absence of galactose. Thus a functional CaGal10 is required not only for galactose metabolism but also for normal hyphal morphogenesis, 
                                    colony morphology, maintenance of cell-wall integrity and for resistance to oxidative stress even in the absence of galactose.</p>
                                
                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/17178245/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/singh.bbrc.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Global analysis of altered gene expression during morphogenesis of Candida albicans in vitro.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Singh V, Sinha I, Sadhale PP. </a>
                                </p>
                                <p><b>Journal: Biochem Biophys Res Commun. 2005 Sep 9;334(4):1149-58. </b></p>    
                                <p> Based on the gene expression profiles and mutant studies we propose thatmorphogenetic response of C. albicans under the conditions used in these experiments is mainly through the pathways controlled by the above two regulators and not through the RIM101-dependent pathway.</p>
                                
                                <p><b>Tools used: Microarray Analysis</b></p>
                                <a class="read-more" href="https://pubmed.ncbi.nlm.nih.gov/16039996/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                               
                            </div>
            
            </div><!-- end of blog wrapper -->
                        <!-- blog-card -->
                        <div class="blog-card">
                            <div class="img-holder">
                                <img src="assets/imgs/sunanda.jpg" alt="Project 2">
                            </div>
                            <div class="content-holder">
                                <h6 class="title">Domainal organization of the lower eukaryotic homologs of the yeast RNA polymerase II core subunit Rpb7 reflects functional conservation.</h6>
            
                                <p class="post-details">
                                    <a href="#">Authors: Singh SR, Rekha N, Pillai B, Singh V, Naorem A, Sampath V, Srinivasan N, Sadhale PP.</a>
                                </p>
                                <p><b>Journal: Nucleic Acids Res. 2004 Jan 2;32(1):201.</b></p>    
                                <p> The subcomplex of Rpb4 and Rpb7 subunits of RNA pol II in Saccharomyces cerevisiae is known to be an important determinant of transcription under a variety of physiological stresses. In S.cerevisiae, RPB7 is essential for cell viability while
                                     rpb4 null strains are temperature sensitive at low and high temperatures. The rpb4 null strain also shows defect in sporulation and a predisposed state of pseudohyphal growth. 
                                     We show here that, apart from S.cerevisiae Rpb7, the Rpb7 homologs from other lower eukaryotes like Schizosaccharomyces pombe, Candida albicans and Dictyostelium discoideum can 
                                     complement for the absence of S.cerevisiae RPB7. </p>
                                

                                <a class="read-more" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC373273/">Check it out on PubMed<i class="ti-angle-double-right"></i></a>
                               
                            </div>
            
           
            </div><!-- end of blog wrapper -->

        </div>
    </div>
</section>

<footer class="footer py-3">
    <div class="container">
        <p class="small mb-0 text-light">
            &copy; <script>document.write(new Date().getFullYear())</script>  email:    vijender.singh@uconn.edu       or        singh.vijender@gmail.com 
        </p>
    </div>
</footer>

<!-- core  -->
<script src="assets/vendors/jquery/jquery-3.4.1.js"></script>
<script src="assets/vendors/bootstrap/bootstrap.bundle.js"></script>

<!-- bootstrap 3 affix -->
<script src="assets/vendors/bootstrap/bootstrap.affix.js"></script>

<!-- Isotope  -->
<script src="assets/vendors/isotope/isotope.pkgd.js"></script>

<!-- Google mpas -->
<script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCtme10pzgKSPeJVJrG1O3tjR6lk98o4w8&callback=initMap"></script>

<!-- JohnDoe js -->
<script src="assets/js/johndoe.js"></script>

</body>
</html>
